client/homebrew/editor/metadataEditor/metadataEditor.jsx

```diff
@@ -363,12 +364,13 @@ const MetadataEditor = createClass({
 
 			{this.renderAuthors()}
 
-			<StringArrayEditor label='invited authors' valuePatterns={[/.+/]}
+			<TagInput label='invited authors' valuePatterns={[/.+/]}
 				validators={[(v)=>!this.props.metadata.authors?.includes(v)]}
 				placeholder='invite author' unique={true}
 				values={this.props.metadata.invitedAuthors}
 				notes={['Invited author usernames are case sensitive.', 'After adding an invited author, send them the edit link. There, they can choose to accept or decline the invitation.']}
-				onChange={(e)=>this.handleFieldChange('invitedAuthors', e)}/>
+				onChange={(e)=>this.handleFieldChange('invitedAuthors', e)}
+				/>
```

Comment:
```suggestion
				onChange={(e)=>this.handleFieldChange('invitedAuthors', e)}/>
```

Reply:
I'm going to leave these as `evt` for now.  At the very end after each "stage" someone can flip these to `e` if really necessary.  I just prefer `evt` as it's easier to ctrl-f, though I understand that `e` is more commonly used throughout the project.